### PR TITLE
feature: added avatar component

### PR DESCRIPTION
**Pull request by aliamaanansari**
feature: added avatar component

### CODE REVIEW

File: home.css [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+p,
+pre {
+  padding: 0;
+  margin: 0;
+}
+.nav-bar {
+  top: 0;
+  position: fixed;
+  width: 100%;
+  background-color: rgb(236, 231, 231);
```
Comment (Sameony) [inline home.css:24]: you can create separate color variables for better readability

File: home.css [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  padding: 1rem 2rem;
+}
+
+.main-div {
+  margin-top: 5rem;
+  display: flex;
+  flex-direction: row;
```
Comment (Sameony) [inline home.css:31]: you dont need to define this property as flex-direction is by default row

File: index.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>UI</title>
```
Comment (Sameony) [inline index.html:7]: give a related/complete title to the webpage

**Review by Sameony — APPROVED** [review]
do follow up on the minor suggestions before merging

File: home.css [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+}
+.box-xxl {
+  width: var(--xxl);
+  height: var(--xxl);
+}
+.avatar-code {
+  background-color: rgb(1, 22, 39);
+  height: 100px;
+  overflow-y: auto;
+  color: white;
```
Comment (rahulcs754) [inline home.css:124]: create variable for color

**Review by rahulcs754 — APPROVED** [review]
try to variable for color and size 
and use prettier to align your code and remove extra space

**Pull request merged**